kyo-scheduler/jvm/src/main/scala/kyo/scheduler/regulator/Admission.scala

```diff
@@ -18,13 +20,8 @@ final class Admission(
 
     @volatile private var admissionPercent = 100
 
-    final private class ProbeTask extends Task {
-        val start = nowMillis()
-        def run(startMillis: Long, clock: InternalClock) = {
-            measure(nowMillis() - start)
-            Task.Done
-        }
-    }
+    private val rejected = new LongAdder
+    private val allowed  = new LongAdder
```

Comment:
I've moved some metrics to fields to inspect them. `UnsafeCounter` doesn't offer an API to obtain the current value since it can be a no-op impl.